client/app/visualizations/table/Renderer.jsx

```diff
@@ -65,6 +65,7 @@ export default function Renderer({ options, data, context }) {
         data-test="TableVisualization"
         columns={tableColumns}
         dataSource={preparedRows}
+        childrenColumnName={null}
```

Comment:
```suggestion
        childrenColumnName={[]}
```

Reply:
@gabrieldutra 

![image](https://user-images.githubusercontent.com/12139186/69497889-6d78f080-0eea-11ea-8a53-968df5d63493.png)

Reply:
:man_facepalming:

Reply:
That worked after I update Antd, maybe they changed it to allow array recently?

Reply:
`childrenName` prop is [declared as optional/nullable](https://github.com/ant-design/ant-design/blob/master/components/table/interface.tsx#L212), but later there are no any conditions - it's just used as is - e.g. `item[this.props.childrenName]` - implicit cast to string. I'll try to find a new solution.